cmd/daprovider/daprovider.go

```diff
@@ -192,15 +213,15 @@ func startup() error {
 			return errors.New("--anytrust.enable is required to start an AnyTrust provider server")
 		}
 
-		if config.Anytrust.ParentChainNodeURL == "" || config.Anytrust.ParentChainNodeURL == "none" {
-			return errors.New("--anytrust.parent-chain-node-url is required to start an AnyTrust provider server")
+		if config.ParentChain.NodeURL == "" {
+			return errors.New("--parent-chain.node-url is required to start an AnyTrust provider server")
```

Comment:
I don't know how many people use these and how problematic of a change that would be. I accept your call.
Maybe we should at least modify the error to also say that anytrust.parent-chain-node-url was renamed to parent-chain.node-url? (also in the other cases)